policy.md

```diff
@@ -0,0 +1,47 @@
+# Policy
+
+TBD: Things to cover in the introduction:
+
+  - Commercial use and re-use (wasn't in original outline, but should
+    be represented in this section).
+
+  - Importance of distinguishing between trademarks, copyrights, and
+    patents.
+
+## Copyright
+
+TBD
+
+## Trademark
+
+TBD
+
+## Patent
+
+TBD
+
+## Commercial Use
+
+TBD
+
+## Security
+
+TBD: Security policy comes from threat modeling, risk assessment, and
+cost/benefit analysis.  In Security, distinguish what's discussed in
+this section from what's discussed in the "Security" subsection in
+Adoptability.
+
+## Data Privacy
+
```

Comment:
The clearer we can be about the recommendations + examples we include, the easier it will be for DPG champions in the government (or the legislative arm) to advocate for these changes

Reply:
Understood.  (And comment moved to become an inline TBD, in [commit 36e40e95](https://github.com/unicef/publicgoods-toolkit/pull/24/commits/36e40e9544f361967b7ff8c5f655aa19111b0472).)